src/main/java/com/puppycrawl/tools/checkstyle/utils/CodePointUtil.java

```diff
@@ -42,4 +42,24 @@ public static boolean isBlank(int... codePoints) {
                 .allMatch(Character::isWhitespace);
     }
 
+    /**
+     * Checks if the given code point array contains only whitespace up to specified index.
+     *
+     * @param codePoints
+     *            array of Unicode code point of string to check
+     * @param index
+     *            index to check up to (exclusive)
+     * @return true if all code points preceding given index are whitespace
+     */
+    public static boolean hasWhitespaceBefore(int index, int...codePoints) {
```

Comment:
This was noticed in other PR, please do `int...codePoints` ->  `int... codePoints` here, then rebase PR at https://github.com/checkstyle/checkstyle/pull/11097 on this branch.

Reply:
Oh, okay I changed at https://github.com/checkstyle/checkstyle/commit/0a362f2930d69cd23e07089cd9de1fb0036b16cf.
Will do the change

Reply:
done

Reply:
Resolved.